File: src/_1ProgrammingBasicsWithJavaMay2023/_03ConditionalStatementsAdvanced/_1Lab/_08CinemaTicket.java
package _1ProgrammingBasicsWithJavaMay2023._03ConditionalStatementsAdvanced._1Lab;

import java.util.Scanner;

public class _08CinemaTicket {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String dayOfWeek = scanner.nextLine();
        switch (dayOfWeek) {
            case "Monday":
            case "Tuesday":
            case "Friday":
                System.out.println(12);
                break;
            case "Wednesday":
            case "Thursday":
                System.out.println(14);
                break;
            case "Saturday":
            case "Sunday":
                System.out.println(16);
                break;
        }
    }
}

File: src/_1ProgrammingBasicsWithJavaScriptNovember2024/_3ConditionalStatementsAdvanced/_1Lab/_08CinemaTicket.js
function cinemaTicket(dayOfWeek) {
    let price = 0;
    if (dayOfWeek === "Monday" || dayOfWeek === "Tuesday" || dayOfWeek === "Friday") {
        price = 12;
    } else if (dayOfWeek === "Wednesday" || dayOfWeek === "Thursday") {
        price = 14;
    } else if (dayOfWeek === "Saturday" || dayOfWeek === "Sunday") {
        price = 16;
    }

    console.log(`${price}`)

}